www/js/theme_editor.js

```diff
@@ -37,10 +39,21 @@ const uploadErrorNotification = (message, timeout = 3000) => {
 };
 
 const imageCheck = (filePath) => {
-  const acceptedExtensions = ['.png', '.PNG', '.jpg', '.JPG', '.jpeg', '.JPEG'];
-  const extension = path.extname(filePath);
+  const acceptedExtensions = ['png', 'jpg'];
+  const acceptedMimeTypes = ['image/png', 'image/jpeg'];
 
-  return acceptedExtensions.includes(extension);
+  try {
+    const buffer = readChunk.sync(filePath, 0, 12);
```

Comment:
there's a bug on line 179 that causes this to fail
```
 sortedFiles.forEach((file) => {
        const fullPath = path.resolve(userBackgroundsPath, file);
        if (imageCheck(file)) {
          recentBgsContainer.appendChild(recentSwatchFactory(fullPath));
        } else {
          fs.unlink(fullPath, (deleteError) => {
            if (deleteError) uploadErrorNotification(`Unable to delete a file. - ${deleteError}`);
          });
        }
      });
```
`imageCheck(file)` should be `imageCheck(fullPath)`.

Reply:
Aah, I see. Thanks 🙏